docs/dev/config/armada/base.yaml

```diff
@@ -17,3 +17,14 @@ auth:
     execute_jobs: ["everyone"]
 pulsar:
   enabled: false
+  URL: "pulsar://pulsar:6650"
```

Comment:
If pulsar is disabled do we need this?

Reply:
Good catch - removed in commit 423a4022.